firmware/src/sensors/sensors_task.cpp

```diff
@@ -346,32 +350,32 @@ void SensorsTask::factoryStrainCalibrationCallback()
 
     for (size_t i = 0; i < 3; i++)
     {
-        calibration_scale_ = raw_value / CALIBRATION_WEIGHT;
+        calibration_scale_ = raw_value / calibration_weight;
         raw_value = strain.get_units(10);
         LOGD("Raw value during calibration: %0.2f", raw_value);
 
         strain.set_scale(calibration_scale_);
         delay(200);
         float calibrated_weight = strain.get_units(10);
 
-        while (abs(calibrated_weight - CALIBRATION_WEIGHT) > 0.25)
+        while (abs(calibrated_weight - calibration_weight) > 0.25)
         {
             // If measured calibrated_weight is more than 10g off from the calibration weight get new reading.
-            if (calibrated_weight < CALIBRATION_WEIGHT - 10 || calibrated_weight > CALIBRATION_WEIGHT + 10)
+            if (calibrated_weight < calibration_weight - 10 || calibrated_weight > calibration_weight + 10)
             {
                 // If this runs for "X" runs it prevents "all" other tasks from running after a while, why??????
                 calibrated_weight = strain.get_units(10);
                 delay(500);
                 continue;
             }
 
-            if (calibrated_weight < CALIBRATION_WEIGHT)
+            if (calibrated_weight < calibration_weight)
             {
-                calibration_scale_ -= 1 * abs((calibrated_weight - CALIBRATION_WEIGHT));
+                calibration_scale_ -= 1 * abs((calibrated_weight - calibration_weight));
```

Comment:
why do we multiply by 1? are you trying to cast to integer?

Reply:
Thats actually a very good question :grimacing:

Reply:
Was a remnant of when i was using a different value as the base increment/decrement 0.5 for example.

Reply:
Do you want to remove it so we avoid bring this around

Reply:
Renived already